packages/slate-editor/src/extensions/embed/components/EmbedMenu.tsx

```diff
@@ -59,11 +60,23 @@ const LAYOUT_OPTIONS: OptionsGroupOption<EmbedNode.Layout>[] = [
     },
 ];
 
+const PRESENTATION_OPTIONS: OptionsGroupOption<EmbedNode.Presentation>[] = [
+    {
+        value: EmbedNode.Presentation.EMBED,
```

Comment:
I suggest using `EmbedNode.TYPE` and `BookmarkNode.TYPE` here. It's more straightforward, and you won't need a new custom enum (located in a surprizing place) :)

Reply:
Something weird happens with mocha when I'm trying to use `EmbedNode.TYPE` in `WebBookmarkMenu`:

```log
@prezly/slate-editor:test: > @prezly/slate-editor@0.94.2 test:unit /home/runner/work/slate/slate/packages/slate-editor
@prezly/slate-editor:test: > mocha --exit --config ../../.mocha/config.cjs './src/**/*.test.{ts,tsx}'
@prezly/slate-editor:test: 
@prezly/slate-editor:test: 
@prezly/slate-editor:test: TypeError: Cannot read properties of undefined (reading 'EmbedNode')
@prezly/slate-editor:test:     at Object.EmbedNode (/home/runner/work/slate/slate/packages/slate-editor/src/extensions/embed/index.ts:21:23)
@prezly/slate-editor:test:     at Object.<anonymous> (/home/runner/work/slate/slate/packages/slate-editor/src/extensions/web-bookmark/components/WebBookmarkMenu.tsx:51:16)
```

I removed the import of `EmbedNode` here:

https://github.com/prezly/slate/pull/490/commits/003d705847d567fc1420a870357bb773ff242fbb

And used `"embed"` directly and the type check command works.

---

Here's a failed build:

https://github.com/prezly/slate/actions/runs/6038947737/job/16386466296

Reply:
Also, I have converted the `BookmarkNode` types to use `namespace`:

https://github.com/prezly/slate/pull/490/commits/8ca298fe8ed6facd6b3c5e6b7075f4a9f9f75007

Reply:
> Also, I have converted the BookmarkNode types to use namespace:

Great! :+1: 

> Something weird happens with mocha when I'm trying to use EmbedNode.TYPE in WebBookmarkMenu:

Circular dependency it is!

So using the node types here is not an option, it seems. :thinking: 

Let's use a local type then? Something like: 

```ts
export type Presentation = 'embed' | 'card';
```